livedata/build.gradle

```diff
@@ -124,7 +124,23 @@ dependencies {
     androidTestImplementation "org.mockito:mockito-core:3.2.4"
     androidTestImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
     androidTestImplementation 'org.amshove.kluent:kluent:1.61'
-
-
 }
 
+afterEvaluate {
```

Comment:
this seems awesome, how does it work? got a good blogpost to read for me?

Reply:
hmm, here is some info about mavenLocal:
https://proandroiddev.com/tip-work-with-third-party-projects-locally-with-gradle-961d6c9efb02
android-related customizations:
https://developer.android.com/studio/build/maven-publish-plugin

Consuming mavenLocal dependency in the target project is the same as any other else, e.g.
``` groovy
dependencies {
    api 'com.github.GetStream:stream-chat-android-livedata:0.6.5-SNAPSHOT'
}
```